### DISCUSSION TITLE
State timeline annotation support

### DISCUSSION THREAD

**Discussion by sthalbauer** [grafana/grafana#40021] [Dashboards, Panels & Visualizations]
Currently annotations are not shown on state timeline. [ver <= 8.1.3]

![image](https://user-images.githubusercontent.com/61388981/136063051-9a13bcb8-13f4-467b-9cb9-0d35e40184fc.png)

It would be great, if "state timeline" panel supports annotations or should this be possible with the current version?

**Comment by benkeil** [comment]
any news?

**Comment by natellium** [comment]
Hey there, according to our docs State timeline supports annotations, according to [this PR](https://github.com/grafana/grafana/pull/47887) since 9.1. I will close this escalation as complete, if you run into any issues you can open a [bug report](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=0-bug-report.yaml&title=Product+Area%3A+Short+description+of+bug). Thanks for the contribution!
